code-challenges.test.js

```diff
@@ -14,38 +14,222 @@
+// Create a function named remainderOfThree and assign array as the parameter
+// Set a variable that filters the array for typeof number
+// Map the filtered array for the remainder of each value when divided by three
+// Call the function and pass in the arguments
+const remainderOfThree = (array) => {
+  filterArray = array.filter(value => typeof value === 'number')
+  return filterArray.map(value => value % 3)
+}
+
+remainderOfThree(hodgepodge1)
+remainderOfThree(hodgepodge2)
+
+//---Second Jest Test---
+
+// PASS  ./code-challenges.test.js
+// hitchhikersGuide
+//   ✓ takes in an array of objects, returns array with sentence (1 ms)
+// remainderOfThree
+//   ✓ takes in a mixed data array and returns an array of only the remainders of the numbers when divided by 3 (1 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       2 passed, 2 total
+// Snapshots:   0 total
+// Time:        0.155 s, estimated 1 s
 
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
+describe("sumCube", () => {
+  it("takes in an array of numbers and returns the sum of all the numbers cubed", () => {
+    expect(sumCube(cubeAndSum1)).toEqual(99)
+    expect(sumCube(cubeAndSum2)).toEqual(1125)
+  })
+})
+
 const cubeAndSum1 = [2, 3, 4]
 // Expected output: 99
 const cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
+//---Initial Jest Test---
+
+// FAIL  ./code-challenges.test.js
+// hitchhikersGuide
+//   ✓ takes in an array of objects, returns array with sentence (1 ms)
+// remainderOfThree
+//   ✓ takes in a mixed data array and returns an array of only the remainders of the numbers when divided by 3
+// sumCube
+//   ✕ takes in an array of numbers and returns the sum of all the numbers cubed
+
+// ● sumCube › takes in an array of numbers and returns the sum of all the numbers cubed
+
+//   ReferenceError: sumCube is not defined
+
+//     167 | describe("sumCube", () => {
+//     168 |   it("takes in an array of numbers and returns the sum of all the numbers cubed", () => {
+//   > 169 |     expect(sumCube(cubeAndSum1)).toEqual(99)
+//         |     ^
+//     170 |     expect(sumCube(cubeAndSum2)).toEqual(1125)
+//     171 |   })
+//     172 | })
+
+//     at Object.expect (code-challenges.test.js:169:5)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 2 passed, 3 total
+// Snapshots:   0 total
+// Time:        0.155 s, estimated 1 s
 
 // b) Create the function that makes the test pass.
+
+// Create a function called sumCube and set array as the parameter
+// Set a variable as the mapped area where each value is cubed
+// Return the new array using reduce to add all the numbers together
+// Call the function and pass in the arguments
+const sumCube = (array) => {
+  cubNum = array.map(value => value ** 3)
+  return (cubNum.reduce((a, b) => a + b))
+}
+
+sumCube(cubeAndSum1)
+sumCube(cubeAndSum2)
```

Comment:
Great use of Map and reduce. I'm wondering why you didn't take a similar approach with the first question? Feel free to hit me up with your thought process on slack.